ui/app/src/components/PluginDetailsView/PluginDetailsView.tsx

```diff
@@ -263,9 +276,10 @@ export default function PluginDetailsView(props: PluginDetailsViewProps) {
             variant="contained"
             color="primary"
             className={classes.useBtn}
+            disabled={!installPermission || isInstalled}
```

Comment:
Let's remove the button for either case.
We should consider adding some sort of badge to signal installed.